src/tamp_improv/benchmarks/blocks2d_env.py

```diff
@@ -46,9 +46,16 @@ def __init__(self, render_mode: str | None = None) -> None:
         self._block_height = 0.2
 
         # Set up initial values for robot and block.
-        self._robot_position = (0.5, 1.0)  # this is the center of the robot
-        self._block_position = (0.0, 0.0)  # this is the center of the block
-        self._gripper_status = 0.0  # the gripper is deactivated
+        self._robot_position = np.array([0.5, 1.0], dtype=np.float32)  # center of the robot
+        self._block_position = np.array([0.0, 0.0], dtype=np.float32)  # center of the block
+        self._gripper_status = np.float32(0.0)  # the gripper is deactivated
+
+        # Set up the target block position.
+        self._target_block_position = np.array([0.5, 0.0], dtype=np.float32)
+
+        # Other parameters.
```

Comment:
good thinking to limit the number of steps. but maybe we'll want to experiment with changing this number later on. how about we use the [TimeLimit](https://gymnasium.farama.org/_modules/gymnasium/wrappers/time_limit/) wrapper from gymnasium instead in this case? (for now, we can just remove `_max_steps` and `_step_count` here.